web-console/old-console/cluster.html

```diff
@@ -59,7 +59,7 @@
             <input type="submit" value="Move" />
           </form>
         </div>
-        <!--<div id="drop">
+        <div id="drop">
```

Comment:
How come you are only removing one side of the comment? I have no idea what this is about but should you also remove the `-->` on line 67?

Reply:
Or is your fix that this comment is unbalanced?

Reply:
It looks like it's a comment inside another comment. Might as well just delete the whole thing, I suppose.

Reply:
@leventov could you delete the whole comment?

Reply:
Deleted.